netsuite/10.0/modules/ROOT/pages/netsuite-troubleshooting.adoc

```diff
@@ -82,6 +83,17 @@ To perform a search for Sales Order objects and other unsupported objects, you c
 * Develop a SuiteScript, deploy it as a RESTlet, and call it from the connector.
 * Create a saved search and call it from the connector.
 
+
+[[triggers-custom-records]]
+== Inability to Set a Custom Record as an Object Type for Triggers
```

Comment:
The problem here is that we don't really use "trigger" in the specialist connector docs. Triggers = Sources (I know...it's not good)